datarobot_provider/example_dags/datarobot_create_execution_environment_dag.py

```diff
@@ -11,25 +11,35 @@
 from airflow.decorators import dag
 
 from datarobot_provider.operators.execution_environment import CreateExecutionEnvironmentOperator
+from datarobot_provider.operators.execution_environment import (
+    CreateExecutionEnvironmentVersionOperator,
+)
 
 
 @dag(
     schedule=None,
-    start_date=datetime(2022, 1, 1),
+    start_date=datetime(2023, 1, 1),
     tags=['example', 'custom model'],
     params={
         "execution_environment_name": "Demo Execution Environment",
         "execution_environment_description": "Demo Execution Environment for Airflow provider",
         "programming_language": "python",
         "required_metadata_keys": [{"field_name": "test_key", "display_name": "test_display_name"}],
+        "docker_context_path": "./datarobot-user-models-master/",
```

Comment:
How does this work in a cloud managed Airflow env? Do those platforms allow you to mount files from some other storage and it can show up as a local path?

Reply:
yes, for cloud managed Airflow there is a automatic mapping to cloud storage to use it as a local path